src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -50,6 +59,33 @@ export const RegistrationSearchBar = () => {
       </Field>
       <RegistrationSortSelector />
 
+      <LoadingButton
+        variant="outlined"
+        startIcon={<FileDownloadIcon />}
+        loadingPosition="start"
+        sx={{ gridArea: 'export' }}
+        loading={isLoadingExport}
+        onClick={async () => {
+          setIsLoadingExport(true);
+          try {
+            const queryParam = new URLSearchParams(window.location.search).get('query') ?? '';
+            const fetchExportData = await fetchRegistrationsExport(queryParam);
+
+            const blob = new Blob([fetchExportData], { type: 'text/csv' });
+            const url = URL.createObjectURL(blob);
+            const link = document.createElement('a');
+            link.download = 'result-export.csv';
```

Comment:
Dato burde kanskje være en del av filnavnet? `result-export-<DATE>.csv` feks

Reply:
Støtter det!

Reply:
Blir fort et grisete filnavn om vi tar med query param da, siden det kan være ganske langt... Feks: https://dev.nva.sikt.no/?query=%28entityDescription.contributors.identity.id%3A%22https%3A%2F%2Fapi.dev.nva.aws.unit.no%2Fcristin%2Fperson%2F1136254%22%29+AND+%28topLevelOrganization.id%3D%22https%3A%2F%2Fapi.dev.nva.aws.unit.no%2Fcristin%2Forganization%2F20754.0.0.0%22%29

Reply:
mulig det blir litt missvisende, at det ser ut som at man har et filter på den datoen... så lar det være for nå. man kan uansett se timestamp ved å sjekke når fila ble lastet ned